apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/useSiteOptions.js

```diff
@@ -47,10 +47,11 @@ export default function useSiteOptions(
 					error: false,
 				} )
 			)
-			.catch( ( { message } ) => {
-				noticeOperations.createErrorNotice( message );
+			.catch( () => {
+				createErrorNotice( `Unable to load site ${ siteOption }.` );
```

Comment:
These will need to be translatable; this should get you started: https://github.com/Automattic/wp-calypso/tree/master/packages/i18n-calypso

Reply:
We're using `import { __ } from '@wordpress/i18n';` aren't we.

Though, I think we may need to double check our webpack plugin to see which one is used.

Reply:
Yes, we are ❇️

Reply:
> Though, I think we may need to double check our webpack plugin to see which one is used.

We're all set. WordPress i18n remains as i18n. There's some minor renaming to use the host package.

https://github.com/Automattic/wp-calypso/blob/e3be3133b7539b8cd8b2f8f5816356a3651c6da0/packages/wordpress-external-dependencies-plugin/index.js#L43